### PR TITLE
Add mro UAVCAN Adapter Node page

### PR DESCRIPTION
**Pull request by Hwurzburg**
@pkocmoud Phillip need some help on line 21 and making sure that, as delivered, it supports all discussed...

### CODE REVIEW

File: common/source/docs/common-mro-uavcan-adapter-node.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+.. image:: ../../../images/mRo-can-node.jpg
+
+
+mRo CAN Node uses open-source code, which utilizes the existing and proven Ardupilot UAVCAN driver library (see :ref:`UAVCAN Adapter Nodes <common-uavcan-adapter-node>` for firmware versions and capabilities). It features a  bootloader for "over CAN" firmware updates and an RTOS running Chibios.
+
+The adapter's CAN Node ID can be set manually or automatically (Not yet supported in stable Ardupilot). The hardware design is open source, so that anyone can contribute to the CAN peripheral world with a proven CAN / MCU interface.
+
+As delivered, it is setup to be plug-and-play as a UAVCAN GPS/Magnetometer/LED/Airspeed by simply plugging in the 6pin GH connector from any mRo or other GPS, and connecting I2C Compasses, RGB LEDs, and/or Airspeed sensors supported by ArduPilot, as desired. 
+
+You can change the firmware to ? in https://firmware.ardupilot.org/AP_Periph if you wish. See the :ref:`SLCAN section of UAVCAN advanced setup<common-uavcan-setup-advanced>` for downloading instructions.
```
Comment (pkocmoud) [inline common/source/docs/common-mro-uavcan-adapter-node.rst:21]: I am not sure what should go here? Maybe, "There are several peripheral focused firmware to choose from at:

**Review by pkocmoud — CHANGES_REQUESTED** [review]
See comment, otherwise it looks good to me.

File: common/source/docs/common-mro-uavcan-adapter-node.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-mro-uavcan-adapter-node.rst]: fixed

**Comment by Hwurzburg** [comment]
merging...

**Pull request merged**